x/ibc/20-transfer/types/keys.go

```diff
@@ -23,7 +23,7 @@ const (
 	QuerierRoute = SubModuleName
 
 	// BoundPortID defines the name of the capability key
-	BoundPortID = "bank"
+	BoundPortID = "bankbankbank"
```

Comment:
?

Reply:
Previously necessary due to identifier validation. Can be reverted.

Reply:
```suggestion
	BoundPortID = "bank"
```